edit.html

```diff
@@ -10,6 +10,7 @@
   <link href="components/styles/menu.css" rel=stylesheet>
   <link href="components/styles/world.css" rel=stylesheet>
   <link href="components/styles/social.css" rel=stylesheet>
+  <link href="components/styles/browse.css" rel=stylesheet>
```

Comment:
Can these CSS files be collapsed? It's already kind of not good that they are separated from the rendering.

Reply:
maybe but not bad to keep separate

Reply:
It is extra load time.

Reply:
I mean... 200ms~ not that big of a deal imo right now, pre sure the JS takes another 7 secs to complete

![xcvbxcvbxcvbxcvb](https://user-images.githubusercontent.com/27584221/95562668-4f689180-09d1-11eb-872e-adef93f17938.PNG)

Reply:
It's different on a Quest tethered to a phone.

Reply:
https://www.npmjs.com/package/uglify-js recommend minifying everything into 1 file then

Reply:
2k line CSS files no fun

Reply:
ill do later as a complete optimization, compression, minify.

Reply:
I would just concat them. But yes.